packages/visx-demo/src/sandboxes/visx-brush/Example.tsx

```diff
@@ -164,6 +199,8 @@ function BrushChart({
           />
         </AreaChart>
       </svg>
+      <button onClick={handleResetClick}>Reset</button>&nbsp;
```

Comment:
thanks for the demo addition! wondering if we could make two minor changes:
- Instead of adding a `1 week` position which differs from `initialBrushPosition`, could we re-use `initialBrushPosition`? In this case we could re-name the button label to `Reset` (since it's the initial position, see next point)
- If we make the above^ change, `Reset` seems to behave more like `Clear` (even though the `Brush` method name is `.reset()`) , could we update to the button label + `handleResetClick` variable name to use `clear`?